docs/authentication/certificate-service-authentication.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Certificate service Authentication
+parent: Authentication
+nav_order: 2
+permalink: /authentication/certificate-service
```

Comment:
/authentication/mtls-authentication or /authentication/certificate-authnetication depends on the title you choose